UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Program.cs

```diff
@@ -96,6 +133,9 @@ private static HostBuilder BuildHostConfiguration()
                  services.AddHttpClient<IFileShareServiceClient, FileShareServiceClient>(client =>
                     client.BaseAddress = new Uri(ConfigurationBuilder["FileShareService:BaseUrl"])
                  );
+                 services.AddHttpClient<IFileShareServiceClient, FileShareServiceClient>(client =>
+                 client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(FulfilmentServiceJob,
+                                            Assembly.GetExecutingAssembly().GetCustomAttributes<AssemblyFileVersionAttribute>().Single().Version)));
```

Comment:
The `HttpClient` config on line 137 has already been configured for `IFileShareServiceClient` on line 133, shouldn't this be combined with that one?

```suggestion
                 services.AddHttpClient<IFileShareServiceClient, FileShareServiceClient>(client =>
                    client.BaseAddress = new Uri(ConfigurationBuilder["FileShareService:BaseUrl"])
                 client.DefaultRequestHeaders.UserAgent.Add(new ProductInfoHeaderValue(FulfilmentServiceJob,
                                            Assembly.GetExecutingAssembly().GetCustomAttributes<AssemblyFileVersionAttribute>().Single().Version)));
```

Reply:
As per the suggestion i have made changes.